infrastructure/fluentbit/fluentbit-configmap.yaml

```diff
@@ -115,7 +120,9 @@ objects:
           # filter logs based on certain keyword(s)
           name   grep
           match  app
-          regex  log ERR
+          Logical_Op or
+          regex log ERR
+          regex log FTL
```

Comment:
I think it would be FAT for Fatal (see https://github.com/serilog/serilog/wiki/Configuration-Basics#minimum-level)